plugins/azure/sqlserver/sqlServerManagedIdentity.js

```diff
@@ -0,0 +1,49 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'SQL Server Managed Identity Enabled',
+    category: 'SQL Server',
+    domain: 'Databases',
+    description: 'Ensure that Azure SQL servers have managed identity enabled.',
+    more_info: 'Enabling managed identities eliminate the need for developers having to manage credentials by providing an identity for the Azure resource in Azure AD and using it to obtain Azure Active Directory (Azure AD) tokens.',
+    recommended_action: 'Enable system or user-assigned managed identities for sql servers.',
+    link: 'https://learn.microsoft.com/en-us/azure/azure-sql/database/authentication-azure-ad-user-assigned-managed-identity?view=azuresql',
+    apis: ['servers:listSql'],
```

Comment:
```suggestion
    apis: ['servers:listSql'],
```
add realtime triggers